src/cigar.c

```diff
@@ -22,4 +29,22 @@ void cigar_to_edits(const char *cigar, char **edits)
     *edits = malloc(no_edits + 1);
 
     // construct the edits sequence
+
+    int n = strlen(cigar);
+    int im_len = 0;
+
+    memset(*edits, 0, no_edits+1);
+
+    for (int i = 0 ; i < n ; i++) {
+        if (i % 2 == 0 ) {
+            im_len = atoi(&cigar[i]);
+        }
+        else {
+            char *c = cigar[i]; 
+            for (int j = 0 ; j < im_len ; j++) {
+               strcat(*edits, &c);
```

Comment:
I don't see how this will compile... `&c` has type `char **` and `strcat` wants a `char *`. But okay.

Anyway, you don't need to use `strcat` for something like this. You want to append to edits, so you move the pointer along.

```c
for (int j = 0; j < im_len; j++)
    *edits++ = cigar[i];
```